src/Form.jsx

```diff
@@ -0,0 +1,95 @@
+import React, { useState } from "react";
+import Input from "./Input";
+import { createNewConference } from "./api";
+import { useHistory } from "react-router-dom";
+
+const Form = () => {
+  let history = useHistory();
+  const [state, setState] = useState({
```

Comment:
`state` should `conference`, this is a variable on state rather than the entire state object. Naming convention here could get confusing.
Then `setConference`